test/models/activation_key_test.rb

```diff
@@ -71,5 +72,30 @@ def setup
       new_key.max_content_hosts = 100
       assert !new_key.valid?
     end
+
+    def test_search_name
+      activation_keys = ActivationKey.search_for("name = \"#{@dev_staging_view_key.name}\"")
+      assert_includes activation_keys, @dev_staging_view_key
+    end
+
+    def test_search_organization_id
+      activation_keys = ActivationKey.search_for("organization_id = \"#{@dev_staging_view_key.organization.id}\"")
+      assert_includes activation_keys, @dev_staging_view_key
+    end
+
+    def test_search_environment
+      activation_keys = ActivationKey.search_for("environment = \"#{@dev_staging_view_key.environment.name}\"")
+      assert_includes activation_keys, @dev_staging_view_key
+    end
+
+    def test_search_content_view
+      activation_keys = ActivationKey.search_for("content_view = \"#{@dev_staging_view_key.content_view.name}\"")
+      assert_includes activation_keys, @dev_staging_view_key
+    end
+
+    def test_search_content_view_id
+      activation_keys = ActivationKey.search_for("content_view_id = \"#{@dev_staging_view_key.content_view.id}\"")
+      assert_includes activation_keys, @dev_staging_view_key
+    end
   end
```

Comment:
Possibly, we haven't done that elsewhere that I am aware of and that might add complexity over time (if someone adds a new fixture that meets the criteria and we are trying to rule out all other AKs or something).